src/main/java/org/opensearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -33,10 +33,7 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
-import com.fasterxml.jackson.annotation.JsonAnyGetter;
-import com.fasterxml.jackson.annotation.JsonAnySetter;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
If you are using Intellij, can you set it to no wildcard imports? This post from SO should help with setup for no wildcard imports: https://stackoverflow.com/a/3348855